src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -535,6 +533,16 @@ private void verifyRequestedAttachingToCase() {
         );
     }
 
+    private void verifySuccessResponseWithAttachToCaseReference(ValidatableResponse response) {
+        JsonPath responseJson = response.extract().jsonPath();
+
+        assertThat(responseJson.getList(RESPONSE_FIELD_ERRORS)).isEmpty();
```

Comment:
I think we also need to add one more assertion for warning ?
```assertThat(responseJson.getList("warnings")).isEmpty();```

Reply:
done